pkg/util/ssh.go

```diff
@@ -31,6 +31,8 @@ import (
 	"os"
 	"time"
 
+	"k8s.io/kubernetes/pkg/util/wait"
```

Comment:
This import introduced an import cycle (the wait package already imports the util package) which is causing the unit tests to fail:

```
+++ [0127 18:07:18] Running tests without code coverage
# k8s.io/kubernetes/pkg/util/wait
import cycle not allowed in test
package k8s.io/kubernetes/pkg/util/wait (test)
	imports k8s.io/kubernetes/pkg/util
	imports k8s.io/kubernetes/pkg/util/wait

FAIL	k8s.io/kubernetes/pkg/util/wait [setup failed]
```